give-a-talk.md

```diff
@@ -1,40 +1,30 @@
 ---
 layout: page
-title: Give a Lightning Talk
+title: We want you to give a talk!
 ---
 {% include JB/setup %}
 
-### We want you to give a talk!
-
-Have you been learning JavaScript and want to spend a couple minutes sharing something you learned?
-
-Did you find a library or discover a JS feature you'd like to share?
-
-Have an interesting method for learning or teaching JavaScript?
+If you have something JavaScript-related to share with your peers that you could say/show in less than 5 minutes, **we want to hear you give a talk**!
 
-Want to share something you made?
+Have you been learning JavaScript and want to spend a couple minutes sharing something you learned? Did you find a library or discover a JS feature you'd like to share? Have an interesting method for learning or teaching JavaScript? Want to share something you made? Want to show off a niche feature of one of your favorite libraries?
 
-Want to show off a niche feature of one of your favorite libraries?
+We also have a wiki page with [talk ideas][1] that you can use for inspiration.
 
-If you have something JavaScript-related to share with your peers that you could say/show in less than 5 minutes, **we want to hear you give a talk**!
 
-### How to give a talk
+#### Submitting your talk
 
-Are you thinking of giving a short lightning talk?
+Once you are ready to submit a talk please email your talk subject to the [SDJS organizers](mailto:sandiegojs-organizers@googlegroups.com).
```

Comment:
I had originally put more avenues of contact but thought might be a little choice paralysis versus a singular option.